content/en/project/fMRI_tactile_word_processing/index.md

```diff
@@ -0,0 +1,80 @@
+---
+type: "project" # DON'T TOUCH THIS ! :)
+date: "2023-06-10" # Date you first upload your project.
+# Title of your project (we like creative title)
+title: "Do you feel the words? An fMRI analysis on tactile vs non-tactile words in Mandarin Chinese"
+
+# List the names of the collaborators within the [ ]. If alone, simple put your name within []
+names: [Yusuke Taira]
+
+# Your project GitHub repository URL
+github_repo: https://github.com/NTUsuke/Yusuke_Taira_project
+
+# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
+website:
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/PSY6983-2021/project_template), click `manage topics`.
+# Please only lowercase letters
+tags: [fMRI,parietal lobe, somatosensory cortex, tactile word]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+
+summary: "Language consists of many thousands of words which differ in meaning and syntactic category. Some words may refer to reletevely touchable, hence concrete aspects of the world while others are abstract in meaning which do not have physical references. But which brain areas are associated with tactile word processing? This project aims to investigate brain activation of participants when listening to tactile vs non-tactile word stimuli. The preliminary result shows that large areas of parietal lobe is particularly activated to tactile words compared to non-tactile words"
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: ""
```

Comment:
Please add a cover image for the project.

You can use the activation map you already uploaded, but you'll need to add the name of the file in this field.

Reply:
I have added the activation image file!